File: php/WeatherApp.php
<?php

class WeatherApp {
  function __construct($api_key)
  {
    $this->api_key = $api_key;
    echo "API key is: " . $api_key . "\n";
  }

  public function get_weather($city)
  {
    echo "Getting weather for " . $city . "\n";
  }
}

File: ruby/WeatherApp.rb
require 'faraday'

class WeatherApp
  def initialize(api_key)
  end
  def instruction_message
    "Please enter a city to get weather data"
  end
  def get_weather(city)
    # response = self.api_conn.get 'q=brooklyn,us'
    # puts response
    "Getting weather for #{city.capitalize}"
  end
end
